CHANGELOG.md

```diff
@@ -1,4 +1,70 @@
-## 2.22.3 (Unreleased)
+## 2.23.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.8**
+- Supported Terraform version: **v1.x**
+
+### Features:
+#### Provider:
+1. Implemented support to ignore selected tag changes across all resources
```

Comment:
Looking at this again, maybe it can be worded better. Maybe:
`Implemented support to ignore changes in selected tag keys across all resources`

@inc1t3Ful what do you think?